templates/docs/whats-new.html

```diff
@@ -0,0 +1,547 @@
+{% extends "_layouts/docs.html" %}
+
+{% block content %}
+
+<h1>  What's new in Vanilla {{versionMinor}} </h1>
+
+<hr>
+
+When we add, make significant updates, or deprecate a component we update their status so that it’s clear what’s available to use. Check back here anytime to see current status information.
+
+<table aria-label="What's new in Vanilla {{versionMinor}}">
+  <thead>
+    <tr>
+      <th style="width: 20%">Component</th>
+      <th style="width: 15%">Status</th>
+      <th style="width: 10%">Version</th>
+      <th style="width: 55%">Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- 3.12.0 -->
+    {% for feature in releaseNotes[0].features %}
+    <tr>
+      <th><a href="{{ feature.url }}">{{ feature.component|safe }}</a></th>
+      <td>
+        {% if feature.status=="New" %}
+        <span class="p-status-label--positive">
+        {% elif feature.status=="Updated" %}
+        <span class="p-status-label--information">
+        {% elif feature.status=="Deprecated" %}
+        <span class="p-status-label--negative">
+        {% endif %}
+        {{ feature.status }}</span>
+      </td>
+      <td>{{ releaseNotes[0].version }}</td>
+      <td>{{ feature.notes | safe }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
+
+<h2>Previously in Vanilla v3</h2>
+
+<table aria-label="Previously in Vanilla v3">
+  <thead>
+    <tr>
+      <th style="width: 20%">Component</th>
+      <th style="width: 15%">Status</th>
+      <th style="width: 10%">Version</th>
+      <th style="width: 55%">Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- 3.11.0 -->
+    {% for version in releaseNotes[1:] %}
+    {% for feature in version.features %}
+
+    <tr>
+      <th><a href="{{ feature.url }}">{{ feature.component|safe }}</a></th>
+      <td>
+        {% if feature.status=="New" %}
+        <span class="p-status-label--positive">
+        {% elif feature.status=="Updated" %}
+        <span class="p-status-label--information">
+        {% elif feature.status=="Deprecated" %}
+        <span class="p-status-label--negative">
+        {% endif %}
+        {{ feature.status }}</span>
+      </td>
+      <td>{{ version.version }}</td>
+      <td>{{ feature.notes | safe }}</td>
+    </tr>
```

Comment:
Not the end of the world but there's a double border between versions
![image](https://user-images.githubusercontent.com/11927929/222413158-42dffcd1-0151-4cce-aa0a-8230fb34870b.png)

Reply:
It might well be a subpixel rounding rendering issue. ignore if it's not an easy fix!